app/src/NativeBase/Containers/ListContainer.tsx

```diff
@@ -261,12 +267,21 @@ const ListContainer = (props: {
   }, [params?.options?.events, params?.search, params?.type])
 
   return (
-    <Theme>
-      <SafeArea>
-        <TopOfApp
-          showSearchButton
-          onSearchButtonPress={() => navigate(screens.search[params.type], '')}
-        />
+    <>
```

Comment:
I think instead of `<>`...`</>` things should be wrapped inside a NativeBase `ScrollView` here https://docs.nativebase.io/scrollview (like in e.g. ProfileContainer)

Reply:
Getting this error if I add scrollView: 

`VirtualizedLists should never be nested inside plain ScrollViews with the same orientation because it can break windowing and other functionality - use another VirtualizedList-backed container instead. `

Seems like it doesn't like `<List>` sitting inside it. Maybe it needs to be something like a sectionList from native base instead? 🤔 https://docs.nativebase.io/section-list

Reply:
Aha, no actually I think the problem is - I'm wrong! 🙂  I think you can just leave this as is.